src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/CaseloadIntegrationTest.kt

```diff
@@ -86,6 +86,22 @@ class CaseloadIntegrationTest : IntegrationTestBase() {
         }
       }
     }
+
+    @Test
+    fun checkDateFormats() {
+      prisonerSearchApiMockServer.stubSearchPrisonersByNomisIds()
+
+      webTestClient.get()
+        .uri(GET_PRISONER)
+        .accept(APPLICATION_JSON)
+        .headers(setAuthorisation(roles = listOf("ROLE_CVL_ADMIN")))
+        .exchange()
+        .expectStatus().isEqualTo(OK.value())
+        .expectHeader().contentType(APPLICATION_JSON)
+        .expectBody()
+        .jsonPath("$.cvl.hardStopDate").value<String> { assertThat(it).matches("\\d{2}/\\d{2}/\\d{4}") }
+        .jsonPath("$.prisoner.dateOfBirth").value<String> { assertThat(it).matches("\\d{4}-\\d{2}-\\d{2}") }
```

Comment:
bit janky that we have two different date formats but didn't want to start moving proxied prisoner offender search methods over to use CVL formatting as that would be loads of frontend changes everywhere.

Kept the CVL fields to follow standard cvl date format... we could really do with just moving everything over!